bench/shallow-copy.js

```diff
@@ -0,0 +1,128 @@
+const Benchmark = require('benchmark');
+const suite = new Benchmark.Suite();
+const { eventToMdTable, H2, createTableHeader } = require('../markdown');
+
+const tableHeader = createTableHeader(['name', 'ops/sec', 'samples']);
+
+const headers = {
+  Accept: '*/*',
+  'Accept-Encoding': 'gzip, deflate, br',
+  'Accept-Language': 'en-US,en;q=0.9',
+  'cache-control': 'no-cache',
+  'CloudFront-Forwarded-Proto': 'https',
+  'CloudFront-Is-Desktop-Viewer': 'true',
+  'CloudFront-Is-Mobile-Viewer': 'false',
+  'CloudFront-Is-SmartTV-Viewer': 'false',
+  'CloudFront-Is-Tablet-Viewer': 'false',
+  'CloudFront-Viewer-Country': 'US',
+  'content-type': '',
+  Host: 'xxxxxx.execute-api.us-east-1.amazonaws.com',
+  origin: 'https://xxxxxx.execute-api.us-east-1.amazonaws.com',
+  pragma: 'no-cache',
+  Referer: 'https://xxxxxx.execute-api.us-east-1.amazonaws.com/prod/',
+  'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36',
+  Via: '2.0 00f0a41f749793b9dd653153037c957e.cloudfront.net (CloudFront)',
+  'X-Amz-Cf-Id': '2D5N65SYHJdnJfEmAV_hC0Mw3QvkbUXDumJKAL786IGHRdq_MggPtA==',
+  'X-Amzn-Trace-Id': 'Root=1-5cdf30d0-31a428004abe13807f9445b0',
+  'X-Forwarded-For': '11.111.111.111, 11.111.111.111',
+  'X-Forwarded-Port': '443',
+  'X-Forwarded-Proto': 'https',
+};
+
+suite
+  .add('spread operator', () => {
+    const result = { ...headers };
+  })
+  .add('spread operator (__proto__: null)', () => {
+    const result = { ...headers, __proto__: null };
+  })
+  .add('spread operator (randomProp: true)', () => {
+    const result = { ...headers, randomProp: true };
+  })
+  .add('structuredClone', () => structuredClone(headers))
```

Comment:
```suggestion
  .add('structuredClone', () => structuredClone(headers))
```

Assign it to a variable to a reliable benchmark

Reply:
I didn't understand, the suggestion is the same of the previous code

Reply:
`const result = structuredClone(headers)`